src/distributors/exponential.ts

```diff
@@ -0,0 +1,122 @@
+import { FakerError } from '../errors/faker-error';
+import type { Distributor } from './distributor';
+import { uniformDistributor } from './uniform';
+
+/**
+ * Creates a new function that generates power-law/exponentially distributed values.
```

Comment:
Maybe we can add a more accessible explanation what this means as well.

```suggestion
 * Creates a new function that generates exponentially distributed values.
 * This means that values on the edges (min or max based on configuration) are more likely to be generated.
```

This needs to be put in the second signature as well.

Reply:
AFAICT this is a power law distribution and not a exponential distribution in the mathematical sense.

> This means that values on one side of the of the value range are far more likely than the values on the other side of the range (depending on configuration).

Or something similar.